app/models/page.rb

```diff
@@ -57,6 +57,10 @@ def meta_tags
     tag_names << plugin_names
   end
 
+  def meta_twitter_handle
+    Settings.meta_twitter_handles[language.code.to_sym]
```

Comment:
No, but I didn't have anywhere that seemed like a good place to stick it - it's just a single method, which meant that making a full service class seemed like overkill. I wanted to test it, which meant that it needed to be somewhere that was easy to test. I needed to repeat it, so just sticking it directly into the slim template seemed like the wrong place.

So yeah, didn't really feel like I had a good spot to drop it. Do you have an idea for a better place?